.github/workflows/continuous-integration.yml

```diff
@@ -55,5 +55,14 @@ jobs:
       - name: Build and test using Gradle and default JDK compiler
         run: ./gradlew --continue --no-build-cache --no-daemon linters javadoc build
         if: matrix.java-compiler != 'ecj'
+      - name: Check for Git cleanliness after build and test
+        run: |
+          if [ -n "$(git status --porcelain)" ]; then
```

Comment:
No way to avoid the copy-paste?  Not at all a huge deal, just curious

Reply:
Best avoided by extracting these few lines into their own shells script. I was being lazy, but yeah, I guess that's worth doing unless we expect to retire Travis CI entirely soon.

Reply:
Previously cloned code extracted into shell script in 1bdda400e939292b29b9a017f46a41e1bc2d9ccb.